testing/tests/DevExpress.ui.widgets/filterBuilder.markup.tests.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
```

Comment:
filterBuilder tests are located in `DevExpress.ui.widgets/filterBuilderParts/` 
Just add the `markupTests.js` and include it in `DevExpress.serverSide/filterBuilder.tests.js` and `DevExpress.ui.widgets/filterBuilder.tests.js`.

Reply:
Ok, I'we corrected tests location

Reply:
I think @DokaRus was talking about extracting existing markup tests instead of creating new ones. I suppose there are a lot of suitable tests here [/DevExpress.ui.widgets/filterBuilderParts/commonTests.js](https://github.com/DevExpress/DevExtreme/blob/18_1/testing/tests/DevExpress.ui.widgets/filterBuilderParts/commonTests.js)

Reply:
Oh, sure. I've extracted markup tests.
Also added "filterHelperTestData.js" with data for filterHelper to require it in both test files.